src/pkg_manager.ts

```diff
@@ -26,104 +26,121 @@ export interface PackageManager {
+    await execWithLog('npm', ['run', script], this.cwd);
+  }
 }
 
 class Yarn implements PackageManager {
   constructor(public cwd: string) {}
 
   async install(packages: JsrPackage[], options: InstallOptions) {
-    const args = ["add"];
+    const args = ['add'];
     const mode = modeToFlag(options.mode);
-    if (mode !== "") {
+    if (mode !== '') {
       args.push(mode);
     }
     args.push(...toPackageArgs(packages));
-    await execWithLog("yarn", args, this.cwd);
+    await execWithLog('yarn', args, this.cwd);
   }
 
   async remove(packages: JsrPackage[]) {
     await execWithLog(
-      "yarn",
-      ["remove", ...packages.map((pkg) => pkg.toString())],
+      'yarn',
+      ['remove', ...packages.map((pkg) => pkg.toString())],
       this.cwd,
     );
   }
+
+  async runScript(script: string) {
+    await execWithLog('yarn', [script], this.cwd);
+  }
 }
 
 class Pnpm implements PackageManager {
   constructor(public cwd: string) {}
 
   async install(packages: JsrPackage[], options: InstallOptions) {
-    const args = ["add"];
+    const args = ['add'];
     const mode = modeToFlag(options.mode);
-    if (mode !== "") {
+    if (mode !== '') {
       args.push(mode);
     }
     args.push(...toPackageArgs(packages));
-    await execWithLog("pnpm", args, this.cwd);
+    await execWithLog('pnpm', args, this.cwd);
   }
 
   async remove(packages: JsrPackage[]) {
     await execWithLog(
-      "yarn",
-      ["remove", ...packages.map((pkg) => pkg.toString())],
+      'yarn',
+      ['remove', ...packages.map((pkg) => pkg.toString())],
       this.cwd,
     );
   }
+
+  async runScript(script: string) {
+    await execWithLog('pnpm', [script], this.cwd);
+  }
 }
 
 export class Bun implements PackageManager {
   constructor(public cwd: string) {}
 
   async install(packages: JsrPackage[], options: InstallOptions) {
-    const args = ["add"];
+    const args = ['add'];
     const mode = modeToFlag(options.mode);
-    if (mode !== "") {
+    if (mode !== '') {
       args.push(mode);
     }
     args.push(...toPackageArgs(packages));
-    await execWithLog("bun", args, this.cwd);
+    await execWithLog('bun', args, this.cwd);
   }
 
   async remove(packages: JsrPackage[]) {
     await execWithLog(
-      "bun",
-      ["remove", ...packages.map((pkg) => pkg.toString())],
+      'bun',
+      ['remove', ...packages.map((pkg) => pkg.toString())],
       this.cwd,
     );
   }
+
+  async runScript(script: string) {
+    await execWithLog('bun', ['run', script], this.cwd);
+  }
 }
 
-export type PkgManagerName = "npm" | "yarn" | "pnpm" | "bun";
+export type PkgManagerName = 'npm' | 'yarn' | 'pnpm' | 'bun';
 
 function getPkgManagerFromEnv(value: string): PkgManagerName | null {
-  if (value.startsWith("pnpm/")) return "pnpm";
-  else if (value.startsWith("yarn/")) return "yarn";
-  else if (value.startsWith("npm/")) return "npm";
-  else if (value.startsWith("bun/")) return "bun";
+  if (value.startsWith('pnpm/')) return 'pnpm';
```

Comment:
Please revert all of these formatting changes. Use `deno fmt` to format the code.

Reply:
@bartlomieju thank your assistance the code is formatted using `deno fmt`